applications/luci-app-cloudflared/Makefile

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI for Cloudflared
+LUCI_DEPENDS:=+cloudflared
+LUCI_PKGARCH:=all
+LUCI_DESCRIPTION:=LuCI support for Cloudflared
+
+PKG_MAINTAINER:=<https://github.com/animegasan>
+PKG_NAME:=luci-app-cloudflared
+PKG_VERSION:=2.0.1-20240113
```

Comment:
the version can't be 2 because there no 1 yet. The version of the app is not related to Cloudflared